Resources/Prototypes/Entities/Objects/Power/powercells.yml

```diff
@@ -28,9 +28,13 @@
   - type: Tag
     tags:
       - DroneUsable
+      - PowerCell
   - type: Appearance
   - type: PowerCellVisuals
   - type: Riggable
+  - type: HitscanBatteryAmmoProvider
+    proto: RedLightLaser
+    fireCost: 50
```

Comment:
i feel like this should be defined on the gun itself instead of the powercell. this would make all battery-powered guns use the same hitscan and do the same damage, which feels wrong.

Reply:
this would need separation from battery ammo provider and battery projectile provider, which seems out of scope

Reply:
i mean calling it out of scope is kinda missing the point. if the goal is to add a battery-powered laser, then the actual mechanics of adding it are pretty much fair game in terms of scope.